File: Krootify.Common/DTOs/Album/AlbumToReturnDTO.cs
using Common.DTOs.Song;
using System;
using System.Collections.Generic;

namespace Common.DTOs
{
    public class AlbumToReturnDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string PhotoUrl { get; set; }
        public ICollection<SongForAlbumAndArtistDTO> Songs { get; set; }
        public string Description { get; set; }
        public DateTime ReleaseDate { get; set; }
    }
}

File: Krootify.Angular/src/app/api/models/album-to-return-dto.ts
/* tslint:disable */
/* eslint-disable */
import { SongForAlbumAndArtistDto } from './song-for-album-and-artist-dto';
export interface AlbumToReturnDto {
  description?: null | string;
  id?: number;
  name?: null | string;
  photoUrl?: null | string;
  releaseDate?: string;
  songs?: null | Array<SongForAlbumAndArtistDto>;
}
